### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by julian-villing**
Refactored Strings into `private static final String` fields named as `strMessageContentCamelCaseWithoutSpacedOrSpecialCharacters`

There are probably methods which don't have direct access to view.getString(R.*)

### CODE REVIEW

File: app/src/main/java/de/pscom/pietsmiet/adapters/CardViewAdapter.java [hunk 1]
```diff
@@ -37,6 +37,10 @@
 
 public class CardViewAdapter extends RecyclerView.Adapter<CardViewHolder> {
 
+    private static final String strOpeningUrlStr = "Opening URL...";
```
Comment (l3d00m) [inline app/src/main/java/de/pscom/pietsmiet/adapters/CardViewAdapter.java:40]: User facing strings in die strings.xml

File: app/src/main/java/de/pscom/pietsmiet/adapters/CardViewAdapter.java [hunk 2]
```diff
@@ -37,6 +37,10 @@
...
+    private static final String strCannotOpenBrowserIntentUrlWas = "Cannot open browser intent. Url was: ";
```
Comment (l3d00m) [inline app/src/main/java/de/pscom/pietsmiet/adapters/CardViewAdapter.java:41]: Logs bitte nicht extrahieren

File: app/src/main/java/de/pscom/pietsmiet/presenter/FacebookPresenter.java [hunk 3]
```diff
@@ -26,33 +26,47 @@
...
+    private static final String strKonntePostNichtParsen = "Konnte post nicht parsen";
+    private static final String strCouldntLoadFacebook = "Couldn't load Facebook";
+    private static final String strFacebookKonnteNichtGeladenWerden = "Facebook konnte nicht geladen werden";
+    private static final String strFrom = "from";
+    private static final String strName = "name";
+    private static final String strMessage = "message";
+    private static final String strCreatedTime = "created_time";
+    private static final String strSimpleDateFormat = "yyyy-MM-dd'T'HH:mm:ssZ";
+    private static final String strId = "id";
+    private static final String strWwwDotFacebookDotCom = "http://www.facebook.com/";
```
Comment (l3d00m) [inline app/src/main/java/de/pscom/pietsmiet/presenter/FacebookPresenter.java:42]: Upper case und richtige Namen bitte: FACEBOOK_BASE_URL zum Beispiel

**Review by l3d00m — CHANGES_REQUESTED** [review]

**Comment by l3d00m** [comment]
? @Gambleon

**Comment by julian-villing** [comment]
! @l3d00m

**Pull request closed without merging**